packages/assets-controllers/src/token-service.ts

```diff
@@ -56,7 +61,11 @@ export async function fetchTokenListByChainId(
   const tokenURL = getTokensURL(chainId);
   const response = await queryApi(tokenURL, abortSignal, timeout);
   if (response) {
-    return parseJsonResponse(response);
+    const result = await parseJsonResponse(response);
+    if (Array.isArray(result) && chainId === ChainId['linea-mainnet']) {
+      return result.filter((elm) => elm.aggregators.includes('lineaTeam'));
```

Comment:
```suggestion
      return result.filter((elm) => elm.aggregators.includes('lineaTeam') || elm.aggregators.length >= 3);
```

What do you think about this? This still includes 100% of whats in the `lineaTeam` list.  But continues allowing tokens that made it into 3 sources.  Otherwise we could be excluding tokens that used to appear.

Reply:
Thank you 🙏  for record, added the OR statement to avoid disappearing tokens that used to appear to users before because they were on +3 lists different than lineaTeam